doc/Publishing.md

```diff
@@ -11,6 +12,32 @@ In order to release one should:
 - tag the published version
 - announce release is done
 
+## Updating the release page on Eclipse Foundation
+
+`One week` before the release, you should update the release page to give a heads-up of what is coming in Theia world,
```

Comment:
The wording at the end of the sentence is a bit off ('head-ups of what is coming in Theia world').
We can simplify it to:

```
`One week` before the release, the project's release page should be updated in order to announce upcoming changes.
```

Reply:
Done